tailwind.config.js

```diff
@@ -0,0 +1,169 @@
+module.exports = {
+  theme: {
+    backgroundOpacity: {
+      0: '0',
+      20: '0.2',
+      30: '0.3',
+      40: '0.4',
+      80: '0.8',
+      100: '1'
+    },
+    borderColor: theme => ({
+      1: theme("colors.border.1"),
+      2: theme("colors.border.2"),
+      3: theme("colors.border.3"),
+      4: theme("colors.border.4"),
+      5: theme("colors.border.5"),
+    }),
+    borderWidth: {
+      0: "0",
+      1: "1px",
+      3: "3px"
```

Comment:
Another discussion we can break out, but this `3px` caught my eye - one could argue that it is a value that should not exist. i.e. since our spacing scale accounts for both positive (width/height) and negative (margin/padding) space this border (positive space) is incongruous.

/cc @bstifle

Reply:
this is only used in notice currently, as far as I can tell, we could probably move that to 4px so it fits in with the spacing scale.

Reply:
On one hand, If you're considering a `4px`, I would also suggest adding a `2px`. I am thinking of things like an active tab or user thumb border where `4px` might be a bit too heavy. At that point we could just inherit the default (and maybe kill it's `8px`).

On the other hand, no intermediate step between `1px` and `4px` might make for a unique design constraint :man_shrugging: 

Either way, as long as we stick to a scale we're headed in a good direction.

Reply:
Modal "with color bar" / alert status bar uses 3px (I believe) and tab active borders are 3px as well. We could just bump everything up to 4px if we want to stick to the scale.